packages/app/ios/Orbit/IngestEventEmitter.h

```diff
@@ -0,0 +1,30 @@
+//
+//  EventEmitter.h
+//  Orbit
+//
+//  Created by Ozzie Kirkby on 2022-05-07.
+//
+
+#ifndef EventEmitter_h
```

Comment:
Note: this is my first time writing any sort of meaningful amount of Objective-C so let me know if anything stands out!